src/uct/api/v2/uct_v2.h

```diff
@@ -132,6 +132,18 @@ typedef enum {
 } uct_md_mem_dereg_field_mask_t;
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief field mask of @ref uct_iface_is_reachable_v2
+ */
+typedef enum {
+    UCT_IFACE_IS_REACHABLE_FIELD_DEV_ADDR        = UCS_BIT(0), /**< dev_addr field */
+    UCT_IFACE_IS_REACHABLE_FIELD_IFACE_ADDR      = UCS_BIT(1), /**< iface_addr field */
+    UCT_IFACE_IS_REACHABLE_FIELD_INFO_STR        = UCS_BIT(2), /**< info_str field */
+    UCT_IFACE_IS_REACHABLE_FIELD_INFO_STR_LENGTH = UCS_BIT(3)  /**< info_str_length field */
```

Comment:
pls rename INFO_STR and INFO_STR_LENGTH to INFO_STRING and INFO_STRING_LENGTH (also the struct fields)

Reply:
fixed